CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Params `salesChannel` on all requests
+### Fixed
```

Comment:
That's a breaking change, when this is fixed we will release this as a new major

```suggestion
### Changed
```

Reply:
Alias, da proxima vez vc pode abrir o PR a partir de uma branch do seu repo? Pq dai eu posso corrigir essas coisas. Quando o PR vem de master eu nao posso mexer.

Reply:
@klzns Beleza mano, nao sabia disso, valeu! Ta feito!